db/migrate/20230206142149_remove_report_inverse_columns_logic_feature_toggle_key.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveReportInverseColumnsLogicFeatureToggleKey < ActiveRecord::Migration[6.1]
+  def change
+    # Remove the feature toggle key for the report inverse columns logic
+    # since it is now enabled by default. This should have been done 
+    # in the PR #9830, but was missed.
+    Flipper.feature(:report_inverse_columns_logic).remove
```

Comment:
Hm, I guess this is fine because this is not app code. But what if we replace Flipper with another library one day or the API changes? Then the migration will fail when updating an old database. I did this manually to observe what it does:
```
pry(main)> Flipper.feature(:report_inverse_columns_logic).remove
  TRANSACTION (0.2ms)  BEGIN
  Flipper::Adapters::ActiveRecord::Feature Load (0.3ms)  SELECT "flipper_features".* FROM "flipper_features" WHERE "flipper_features"."key" = $1  [["key", "report_inverse_columns_logic"]]
  Flipper::Adapters::ActiveRecord::Feature Destroy (0.3ms)  DELETE FROM "flipper_features" WHERE "flipper_features"."id" = $1  [["id", 4]]
  Flipper::Adapters::ActiveRecord::Gate Load (0.3ms)  SELECT "flipper_gates".* FROM "flipper_gates" WHERE "flipper_gates"."feature_key" = $1  [["feature_key", "report_inverse_columns_logic"]]
  Flipper::Adapters::ActiveRecord::Gate Destroy (0.5ms)  DELETE FROM "flipper_gates" WHERE "flipper_gates"."id" = $1  [["id", 4]]
  Flipper::Adapters::ActiveRecord::Gate Destroy (0.3ms)  DELETE FROM "flipper_gates" WHERE "flipper_gates"."id" = $1  [["id", 5]]
  Flipper::Adapters::ActiveRecord::Gate Destroy (0.2ms)  DELETE FROM "flipper_gates" WHERE "flipper_gates"."id" = $1  [["id", 6]]
  TRANSACTION (6.9ms)  COMMIT
  Flipper feature(report_inverse_columns_logic) adapter(active_record) remove  (18.2ms)  [ result=true ]
  Flipper feature(report_inverse_columns_logic) remove true (18.3ms)  [ thing=nil ]
=> true
```

Performance doesn't really matter here but it would also improve if we specified the statements explicitly.
```suggestion
    execute("DELETE FROM flipper_gates WHERE feature_key = 'report_inverse_columns_logic'")
    execute("DELETE FROM flipper_features WHERE key = 'report_inverse_columns_logic'")
```